service/key/key.go

```diff
@@ -19,3 +19,7 @@ func ToCluster(obj interface{}) (*v1alpha2.Cluster, error) {
 func Namespace(cluster *v1alpha2.Cluster) string {
 	return fmt.Sprintf("%s-prometheus", cluster.GetName())
 }
+
+func Secret() string {
+	return "cluster-certificates"
```

Comment:
Shouldn't this be `$CLUSTER_ID-prometheus`?

Reply:
Or is this the name of the target secret?

Reply:
That's the name of the secret we are going to create. Just made this name up. I can change this.

Reply:
yeah whatever, this is super ez to change

Reply:
I thought `cluster-certificates` is a meaningful name, tells you that this secret hold the certificates for the cluster. And it would be the same for every cluster, so easy to find.

Reply:
I love it, let's roll

Reply:
#nobikeshedding